documentdata.py

```diff
@@ -0,0 +1,19 @@
+class DocumentData:
+
+    def __init__(self, name, path, type, lang, size, createdAt):
+        self.name = name
+        self.path = path
+        self.type = type
+        self.lang = lang
+        self.size = size
+        self.createdAt = createdAt
+
+    def asDict(self):
```

Comment:
```suggestion
    def as_dict(self):
```
No camel casing in python :)